src/components/Table/Controls.module.scss

```diff
@@ -0,0 +1,11 @@
+.Container {
+    display: flex;
+    justify-content: flex-end;
+    gap: 1rem;
+
+    height: 40px;
```

Comment:
nit: could we switch `px` to `rem`? 

`2.5rem`

Reply:
Good spot, thanks!